ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,842 @@
+{
```

Comment:
<p>You <em>should</em> be able to just call <code>settings</code> here and get the pprint dictionary output.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>